coveralls/cli.py

```diff
@@ -19,7 +19,9 @@
 Global options:
     --service=<name>  Provide an alternative service name to submit.
     --rcfile=<file>   Specify configuration file. [default: .coveragerc]
+    --base_dir=<dir>  Base directory that is removed from reported paths.
```

Comment:
Please rename to:
```suggestion
    --basedir=<dir>   Base directory that is removed from reported paths.
```

I was going to think through a convention for hyphens vs underscores for this, only to realize that was a silly level of overkill when we can just side-step any future consistency issues.